sklego/ensemble_model.py

```diff
@@ -0,0 +1,87 @@
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix
+
+class EnsembleModel():
+    """
+    A classifier that takes the uncertainty p(class | model) of a classifier into account.
+    Based on the requirements of issue #23 of koaning/scikit-lego
+    """
+
+    def __init__(self, X_test, y_test, estimator=LogisticRegression(), alpha=0.5):
```

Comment:
So ... 

... I think it might be a good idea to have a seperate validation set on which to base the confusion matrix. However this object needs to primarily fit a `sklearn.pipeline` so the base behavior should be to base the confusion matrix on the training set. 

I also think it might be nicer not to use `LogisticRegression()` as a base input and instead demand that the user defines this explicitly. Explicit > Implicit.

Reply:
Also this might be a better order. 

```
def __init__(self, estimator=LogisticRegression(), alpha=0.5, X_test=None, y_test=None)
```